Worksheet_B/Pseudocode

```diff
@@ -0,0 +1,15 @@
+Open Terminal
```

Comment:
The worksheet required you to write pseudocode for the similarity score calculation, i.e. to count the number of correct letters in the guessed word. Make sure you read the worksheet carefully in future!